### ISSUE TITLE
Xcode 15 beta 5 - Multiple errors

### ISSUE THREAD

**Issue by tiagosito** [jerson/flutter-rsa#55]
### Description
Using Xcode 15 beta 5, I get the following error for lib **fast_rsa**:

**Environment**
Flutter version: 3.7.12
fast_rsa version: 3.5.7 or below
With mac Os chip M2

**When trying to build the project**
```
ld: warning: '../fast_rsa/ios/librsa_bridge.a[x86_64][2]( go.o )' has malformed LC_DYSYMTAB, expected 52 undefined symbols to start at index 6997, found 64 undefined symbols starting at index 56
ld: warning: '../fast_rsa/ios/librsa_bridge.a[x86_64][2](go.o)' has malformed LC_DYSYMTAB, expected 52 undefined symbols to start at index 6997, found 64 undefined symbols starting at index 56

ld: multiple errors: initializer pointer must point to start of function (no addend) in '../fast_rsa/ios/librsa_bridge.a[x86_64][2](go.o)'; initializer pointer must point to start of function (no addend) in '../fast_rsa/ios/librsa_bridge.a[x86_64][2](go.o)'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I found this issue related but the proposed solution didn't work for me [issue related Xcode 15 beta](https://github.com/golang/go/issues/60694)

**Comment by jerson** [comment]
Hi @tiagosito could you test with this branch, I have updated the binaries built with the latest toolkit

https://github.com/jerson/flutter-rsa/pull/56

**Comment by tiagosito** [comment]
Hi @jerson, 

I tried the solution with the new branch, I did all the work to clean the caches and pods and only then get everything updated, but it didn't result in a different output from the previous one.

Also I tried to run with a new flutter example project and just added the fast_rsa lib, so I got the same error

```
ld: warning: '../ios/librsa_bridge.a[x86_64][2](go.o)' has malformed LC_DYSYMTAB, expected 59 undefined symbols to start at index 6924, found 68 undefined symbols starting at index 56
ld: warning: '../ios/librsa_bridge.a[x86_64][2](go.o)' has malformed LC_DYSYMTAB, expected 59 undefined symbols to start at index 6924, found 68 undefined symbols starting at index 56

ld: multiple errors: initializer pointer must point to start of function (no addend) in '../ios/librsa_bridge.a[x86_64][2](go.o)'; initializer pointer must point to start of function (no addend) in '../ios/librsa_bridge.a[x86_64][2](go.o)'
clang: error: linker command failed with exit code 1 (use -v to see invocation)

```

**Comment by jerson** [comment]
I'm testing with go1.21.0 and is working fine, going to deploy a new version in some mins

**Comment by jerson** [comment]
at this point I think is a good chance to drop armv7 support, I'm not sure what is the situation with armv7 is still a thing do you know ?

**Comment by jerson** [comment]
Hi @tiagosito could you test using the new version please https://pub.dev/packages/fast_rsa/versions/3.6.0

**Comment by tiagosito** [comment]
I'm closing the issue because the [new published version 3.6.0](https://pub.dev/packages/fast_rsa/versions/3.6.0) solved the reported problem.
